src/Symfony/Component/Translation/Dumper/JsonFileDumper.php

```diff
@@ -23,9 +23,23 @@ class JsonFileDumper extends FileDumper
     /**
      * {@inheritdoc}
      */
+    protected function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
+    {
+        $flags = isset($options['flags']) ? (int) $options['flags'] : 0;
```

Comment:
This is wrong, the variable name must be `$flags` but the option name must be "json_encoding"